irods_store.py

```diff
@@ -397,7 +412,7 @@ def delete(self, location):
 
```

Comment:
Another thing worth mentioning is how we changed the logging level of the PythonIrodsClient and maybe include a sample __init__.py file otherwise the developer can run into logging errors